store/sql_store.go

```diff
@@ -110,6 +111,14 @@ func initConnection() *SqlStore {
 		}
 	}
 
+	if len(utils.Cfg.SqlSettings.DataSourceSearchReplica) == 0 {
+		sqlStore.searchReplica = sqlStore.master
```

Comment:
I think those are pretty much equivalent. So whichever one works better in the code.